src/Database/NeoServer.Data/Contexts/NeoContext.cs

```diff
@@ -67,6 +68,44 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
         modelBuilder.ApplyConfiguration(new GuildMembershipEntityConfiguration());
         modelBuilder.ApplyConfiguration(new PlayerStorageEntityConfiguration());
 
+        foreach (var entity in modelBuilder.Model.GetEntityTypes())
```

Comment:
I think we can remove this and rename the tables manually to lower case and using underscore "_" on compound noun
ex:
PlayerItem -> player_item